packages/xrpl/HISTORY.md

```diff
@@ -52,80 +55,6 @@ Bundler configurations are much more simplified. See [../UNIQUE_STEPS](Unique St
 * Deprecated:
   * `convertHexToString` in favor of `@xrplf/isomorphic/utils`'s `hexToString`
   * `convertStringToHex` in favor of `@xrplf/isomorphic/utils`'s `stringToHex`
-  
```

Comment:
I think you accidentally deleted some HISTORY.md here (Specifically the beta notes)

Reply:
That was on purpose. You don't need the beta logs in the HISTORY after the new version is released, since it's not an official release.

Reply:
I see, ok.